externtype.go

```diff
@@ -46,7 +46,7 @@ func (ty *ExternType) FuncType() *FuncType {
 	return mkFuncType(ptr, ty.owner())
 }
 
-// Returns the underlying `GlobalType` for this `ExternType` if it's a function
+// GlobalType returns the underlying `GlobalType` for this `ExternType` if it's a function
```

Comment:
Looks like an original copy/paste error. Pretty sure this should read:

    GlobalType returns the underlying `GlobalType` for this `ExternType` if it's a *global* type. Otherwise returns `nil`.

Reply:
This goes for `TableType` and `MemoryType` below as well. They all say they return the underlying X if it's a function.